platform/ui/src/components/SegmentationGroupTable/SegmentationConfig.tsx

```diff
@@ -78,8 +90,11 @@ const ActiveSegmentationConfig = ({
         <InputRange
           minValue={0}
           maxValue={usePercentage ? 100 : 1}
-          value={usePercentage ? config.fillAlpha * 100 : config.fillAlpha}
+          value={
+            useFillAlphaPercentage ? config.fillAlpha * 100 : config.fillAlpha
```

Comment:
As per my earlier comment... initially here config.fillAlpha should indeed be multiplied by 100. However, once the onChange is executed below (via a change from the InputRange) it should no longer be multiplied by 100. I am tracking this association locally in this component now, but that does not feel 100% correct to me. Thoughts?